homu/html/queue.html

```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="utf-8">
         <title>Homu queue - {{repo_name}}</title>
+        <link href="//cdn.datatables.net/1.10.4/css/jquery.dataTables.min.css">
```

Comment:
Huh, does it? Indeed, styles will not be that significant considering the current usage in the code. Maybe we can remove it entirely? :trollface: